DuckDuckGo/TabViewController.swift

```diff
@@ -774,10 +774,6 @@ extension TabViewController: WKNavigationDelegate {
             return .cancel
         }
         
-        guard let documentUrl = navigationAction.request.mainDocumentURL else {
-            return .allow
-        }
```

Comment:
As I understand `documentUrl` is not needed anymore, but this change does not only remove unnecessary variable but also changes the app logic. Please keep the guard and just tweak the condition.